ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import * as Types from '@types';
+import * as Interfaces from '@interfaces';
+
+type Props = {
+    publicationData: Interfaces.Publication;
+    user: Types.UserType;
+};
+
+const ApprovalsTracker: React.FC<Props> = (props): React.ReactElement => {
+    return (
+        <>
+            <p className="my-2 text-lg dark:text-white-50">
+                Your role on this publication:{' '}
+                {props.user.id === props.publicationData.createdBy ? 'Corresponding author' : 'Author'}
+            </p>
+            <div className="my-2 overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 dark:ring-transparent">
+                <table
+                    data-testid="approval-tracket-table"
+                    className="mb- min-w-full divide-y divide-grey-100 dark:divide-teal-300"
+                >
+                    <thead className="bg-grey-50 transition-colors duration-500 dark:bg-grey-700">
+                        <tr>
+                            <th className="whitespace-pre py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-grey-900 transition-colors duration-500 dark:text-grey-50 sm:pl-6 ">
+                                Author
+                            </th>
+                            <th className="whitespace-pre py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-grey-900 transition-colors duration-500 dark:text-grey-50 sm:pl-6 ">
+                                Status
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody className="divide-y divide-grey-100 bg-white-50 transition-colors duration-500 dark:divide-teal-300 dark:bg-grey-600">
+                        {props.publicationData.coAuthors.map((CoAuthor) => (
+                            <tr key={CoAuthor.id}>
+                                <td className="space-nowrap py-4 pl-4 pr-3 text-sm text-grey-900 transition-colors duration-500 dark:text-white-50 sm:pl-6">
+                                    <div className="space-y-2">
+                                        {CoAuthor.linkedUser ? (
+                                            <p>
+                                                {props.publicationData.createdBy === CoAuthor.linkedUser
+                                                    ? `${CoAuthor.user?.firstName} ${CoAuthor.user?.lastName} (Corrosponding Author)`
+                                                    : `${CoAuthor.user?.firstName} ${CoAuthor.user?.lastName} `}
```

Comment:
*Corresponding

Reply:
'Corresponding Author' is displayed on the 'Status' column, we don't need to display it here also

Reply:
On the mockup, there's a small "(you)" text next to the current logged in user name
![image](https://user-images.githubusercontent.com/48569671/218040630-8b10ae69-4371-4fab-a930-02bcb3c37751.png)